Makefile

```diff
@@ -128,6 +128,7 @@ api-codegen:
 	@python -m venv env
 	@. env/bin/activate
 	@pip install -q -e .
+	@pip install -q -e .[reader]
```

Comment:
Need to install reader package.

![image](https://github.com/ansys/pyfluent/assets/106588300/f37aa142-a377-41ef-996d-e155a738b043)

https://github.com/ansys/pyfluent/actions/runs/9889959134/job/27317176855?pr=3058

Reply:
This is because CaseReader is now a top-level import. I think we should avoid that as the reader package is optional.

Or, maybe we can include the h5py dependency by default. We had separated it due to some temporary issue in a previous h5py version. TBD.

Reply:
Ok. We will add `h5py` dependency by default.